src/services/userService.js

```diff
@@ -35,3 +36,36 @@ export function getUserProfileInfo() {
             );
         });
 }
+export function editUserInfo(values) {
+    let model = {
+        name: values.name,
+        surname: values.surname,
+        email: values.email
+    };
+
+    userService
+        .editUserInfo(model)
```

Comment:
You can simply send _values_ without creating a _model_, because they have the same fields